### PR TITLE
SW-1014 added migration step

### PR DESCRIPTION
**Pull request by JFSolution**
changed logrotate files
migration has to be tested

### CODE REVIEW

**Comment by JFSolution** [comment]
why this doesn't run the sonar cloud analysis??

**Comment by JFSolution** [comment]
> why this doesn't run the sonar cloud analysis??

ahh the workflow is not defined on this branch

File: octoprint_mrbeam/migration/Mig003.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        ]
+        for logrotate in logrotates:
+            self._logger.debug("enable logrotate of " + logrotate)
+            src = os.path.join(
+                __package_path__, self.MIGRATE_LOGROTATE_FOLDER, logrotate
+            )
+            dst = os.path.join("/etc/logrotate.d/" + logrotate)
+            self.exec_cmd("sudo cp {src} {dst}".format(src=src, dst=dst))
+
+        self._logger.debug("restart of logrotate so changed config will take affect")
```
Comment (Yelko-mrbeam) [inline octoprint_mrbeam/migration/Mig003.py]: Grammar Nazi coming through: -> restarting logrotate in order for the changed config to take **effect**

That's what's happening right?

File: tests/migrations/test-migration-Mig003.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from octoprint_mrbeam.migration import Mig003EnableLogrotateBuster
+import unittest
+
+
+class TestMigrationMig003(unittest.TestCase):
```
Comment (Yelko-mrbeam) [inline tests/migrations/test-migration-Mig003.py:5]: 👏 I love it

**Review by Yelko-mrbeam — CHANGES_REQUESTED** [review]

**Review by Yelko-mrbeam — APPROVED** [review]

**Pull request merged**